Dockerfile

```diff
@@ -47,9 +49,11 @@ RUN cmake --build . --config Release -j8
 
 FROM alpine
 
+RUN addgroup -S ucentralfms && adduser -S -G ucentralfms ucentralfms
+
 RUN mkdir /ucentral
 RUN mkdir /ucentralfms-data
-RUN apk add --update --no-cache librdkafka curl-dev 
+RUN apk add --update --no-cache librdkafka curl-dev mariadb-connector-c libpq unixodbc 
```

Comment:
IMO we should put this in different PR as it is clearly doesn't fit into scope of this PR

Reply:
agree, reverted the changes and will put them in a separate PR.